autoload/gitmessenger/blame.vim

```diff
@@ -430,7 +430,7 @@ function! gitmessenger#blame#new(file, line, opts) abort
     let b.opts = a:opts
 
     let dir = fnamemodify(a:file, ':p:h')
-    let b.git_root = gitmessenger#git#root_dir(dir)
+    let b.git_root = fnamemodify(system('git rev-parse --show-toplevel')[:-2], ':p')
```

Comment:
This implementation does not work because `git` command finds `.git` directory based on current working directory. It must be searched in the `dir` directory. For example, when running `:GitMessenger` from outside Git repository for the file which is managed in Git repo, it does not work.